package.json

```diff
@@ -18,7 +18,7 @@
     "build:production": "ember build --environment=production",
     "prepublish": "bower install && npm run-script build:production",
     "start": "ember serve",
-    "test": "testem -R dot ci",
+    "test": "node_modules/broccoli-jscs/node_modules/jscs/bin/jscs packages && testem -R dot ci",
```

Comment:
@rwjblue this seems a little extreme, but `jscs` isn't a direct dependency.

Is this better or worse than adding `jscs` to the dependencies and executing it directly?

Should I add `npm install -g jscs` to `.travis.yml`?